examples/mplot3d/rotate_axes3d_sgskip.py

```diff
@@ -5,7 +5,7 @@
 
 A very simple animation of a rotating 3D plot about all 3 axes.
 
-See wire3d_animation_demo for another simple example of animating a 3D plot.
+See :doc:`wire3d_animation_demo` for another example of animating a 3D plot.
```

Comment:
You need to specify the relative path /gallery/section/file

https://matplotlib.org/devdocs/devel/documenting_mpl.html#referring-to-other-documents-and-sections